code/src/WijmoProvider/Columns/DropdownColumn.ts

```diff
@@ -169,7 +179,9 @@ namespace WijmoProvider.Column {
                     this.config.dropdownOptions = values;
                     dataMap.collectionView.sourceCollection = values;
                     dataMap.collectionView.refresh();
-                    if (this.config.parentBinding) {
+
+                    // eslint-disable-next-line no-extra-boolean-cast
+                    if (!!this.config.parentBinding) {
```

Comment:
What does this mean? (!!)

Reply:
```
It basically checks if the variable in question has a value, making a cast to boolean.

- "",  0, null, undefined and NaN will always return False.
- {}, [], "string", 999, new Date() will always return True.
```

I'm not sure about using it here. But anyway, it's checking if the **parentBinding** is neither undefined nor empty.